src/web/nextui/src/app/eval/ResultsTable.tsx

```diff
@@ -303,29 +303,45 @@ function EvalOutputCell({
     } catch (error) {
       console.error('Invalid regular expression:', (error as Error).message);
     }
-  } else if (renderMarkdown) {
-    node = (
-      <ReactMarkdown
-        components={{
-          img: ({ src, alt }) => (
-            <img
-              loading="lazy"
-              src={src}
-              alt={alt}
-              onClick={() => toggleLightbox(src)}
-              style={{ cursor: 'pointer' }}
-            />
-          ),
-        }}
-      >
-        {text}
-      </ReactMarkdown>
-    );
-  } else if (prettifyJson) {
-    try {
-      node = <pre>{JSON.stringify(JSON.parse(text), null, 2)}</pre>;
-    } catch (error) {
-      // Ignore because it's probably not JSON.
+  } else {
+    if (prettifyJson) {
+      try {
+        let json = JSON.parse(text);
+        if (jsonExtractField) {
+          json = json[jsonExtractField];
+          if (!json) {
+            node = (
+              <em>
+                Field <code>{jsonExtractField}</code> is undefined
+              </em>
+            );
+          }
+        }
+        if (json) {
+          node = <pre>{JSON.stringify(json, null, 2)}</pre>;
```

Comment:
Any way that the extracted field could just be rendered the same way as if was just text? That is, wrapped in `<ReactMarkdown>`?